### ISSUE TITLE
Missing `<main>` element on patient self registration

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4111]
## Area or Component
Patient self-registration form
Example: https://dev.simplereport.gov/app/register/E8Q2P

## Quick Summary
There's no `<main>` element on the page, so most of the page is not covered by landmarks.

## Accessibility Violation
All page content should be contained by landmarks: https://dequeuniversity.com/rules/axe/4.4/region?application=AxeChrome
Document should have one main landmark: https://dequeuniversity.com/rules/axe/4.4/landmark-one-main?application=AxeChrome

## Action Requested
Add a `<main>` landmark to the patient self registration page

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=102767267&range=61:61
